gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-sync/src/main/java/io/gravitee/gateway/services/sync/process/common/deployer/SubscriptionDeployer.java

```diff
@@ -70,6 +78,12 @@ public Completable deploy(final SubscriptionDeployable deployable) {
                     .subscriptions()
                     .stream()
                     .filter(subscription -> deployable.subscribablePlans().contains(subscription.getPlan()))
+                    .sorted((o1, o2) -> {
```

Comment:
Instead of having a hard code rules, I would apply the same `sort` as it exists on the SubscriptionSynchronizer by updating the SubscriptionAppender with the following code l85.
```
return subscriptionRepository
                .search(criteriaBuilder.build(),
                     new SortableBuilder().field("updatedAt").order(Order.ASC).build())
                .stream()
```